package/src/languageService/settings.ts

```diff
@@ -18,6 +18,7 @@ export interface LanguageSettings {
     quickFixCodeActions?: QuickFixCodeActionOptions[];
     enableQuickFixes?: boolean;
     documentationBaseUrl?: boolean;
+    documentationSuffix?: string;
```

Comment:
Might be worth adding a doc comment here

Reply:
Where exactly? Knowhere else has

Reply:
```suggestion
    /**
     *  Enables god mode
     */
    documentationSuffix?: string;
```